src/persistent_stores/store_instances/config_store/store_definition.hpp

```diff
@@ -61,6 +61,8 @@ struct CurrentStore : public journal::CurrentStoreConfig<journal::Backend, backe
     StoreItem<std::array<char, lan_hostname_max_len + 1>, defaults::net_hostname, journal::hash("LAN Hostname")> lan_hostname;
 
     StoreItem<int8_t, defaults::lan_timezone, journal::hash("LAN Timezone")> timezone; // hour difference from UTC
+    StoreItem<time_tools::TimeOffsetMinutes, defaults::lan_timezone_min, journal::hash("LAN Timezone Minutes")> timezone_min; // minutes offset for hour difference from UTC
```

Comment:
I'm afraid the `lan_` prefix (both in the name and the default) is wrong (this has nothing to do with Local Area Network configuration). This mistake slipped through the migration and since its hash is used to identify it in the EEPROM, we can't change it now.

But I would like to avoid bringing the same error into the new items.

Reply:
I've removed the LAN prefix from both variable names and hash names as you recommended.